tests/test_mirrors.py

```diff
@@ -80,31 +111,13 @@ def test_get_list_of_mirrors(self):
     self.assertTrue(self.mirrors['mirror4']['url_prefix']+'/metadata/release.txt' in \
                     mirror_list)
 
-    # 1 match: a mirror without target directory confinement
-    mirror_list = mirrors.get_list_of_mirrors('target', 'a.txt', self.mirrors)
-    self.assertEqual(len(mirror_list), 1)
-    self.assertTrue(self.mirrors['mirror1']['url_prefix']+'/targets/a.txt' in \
-                    mirror_list)
-
-    mirror_list = mirrors.get_list_of_mirrors('target', 'a/b', self.mirrors)
-    self.assertEqual(len(mirror_list), 1)
-    self.assertTrue(self.mirrors['mirror1']['url_prefix']+'/targets/a/b' in \
-                    mirror_list)
-
     # No matches
     mirror1 = self.mirrors['mirror1']
     del self.mirrors['mirror1']
+    del self.mirrors['mirror5']
     mirror_list = mirrors.get_list_of_mirrors('target', 'a/b', self.mirrors)
     self.assertFalse(mirror_list)
-    self.mirrors['mirror1'] = mirror1
```

Comment:
Deleted this line because the tests below don't actually care about the content of `self.mirrors`.